.github/workflows/sphinx.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Install Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.10
+          python-version: "3.10"
```

Comment:
Fix templating is broken, I wouldn't spend too much time on updating it: https://github.com/lyft/cartography/issues/1284